acceptance/ui/features/hosts.feature

```diff
@@ -18,15 +17,204 @@ Feature: Host Management
       And I should see "Showing 0 Results"
 
   @login-required
-  Scenario: Add a host
+  Scenario: View Add Host dialog
     When I am on the hosts page
       And I click the Add-Host button
-    Then I should see "Host and port"
+    Then I should see the Add Host dialog
+      And I should see "Host and port"
+      And I should see the Host and port field
       And I should see "Resource Pool ID"
-    When I fill in the Host Name field with "gjones-dev:4979"
+      And I should see the Resource Pool ID field
+      And I should see "RAM Commitment"
+      And I should see the RAM Commitment field
+
+  @login-required @emptyHostsPage
+  Scenario: Add an invalid host with an invalid name
```

Comment:
Please add another Scenario for a similar, but slightly different case, and that is a valid hostname (something that's in the host table) and port number, but the address/port combination does not match a running instance of CC.  This case will also fail, but the error message is different.   For instance, in your tests, vagrant:4979 is a valid CC host, so try something like vagrant:9999 - in this case, the hostname "vagrant" is a valid host name (it's in /etc/hosts), but there is no CC instance running on that host at port 9999